rest-server/src/models/job.js

```diff
@@ -37,7 +37,7 @@ class Job {
       for (let key of Object.keys(job)) {
         this[key] = job[key];
       }
-      next();
+      next(this);
```

Comment:
Will the constructor function return AFTER this next(this) function call returns? If not, the ‘undefined’ issue may still be there.

Reply:
OK. I found the 'load' function calls 'next()' when Job's constructor function is done: https://github.com/Microsoft/pai/blob/def8a2a7b8cf2a22aa7e65dfec72f805dc88f20e/rest-server/src/controllers/job.js#L41. So this method works correctly.

Reply:
it uses aync callback instead of sync return